examples/kube-trigger/sample.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: this-will-trigger-update-1
```

Comment:
Definition has a `example-url`  feature, let's have a try for this definition. refer to: https://github.com/kubevela/kubevela/pull/4377.

Reply:
let's leave it in another PR, I'll merge and try this addon